sklearn/preprocessing/_data.py

```diff
@@ -342,8 +352,13 @@ def partial_fit(self, X, y=None):
             The data used to compute the mean and standard deviation
             used for later scaling along the features axis.
 
-        y
-            Ignored
+        y : None
+            Ignored.
+
+        Returns
+        -------
+        self : object
```

Comment:
```suggestion
        self : object
```

Reply:
not sure I can see any difference in the existing and suggested change. Am I missing something?

Reply:
I think it was the wrong line